src/fastly/vcl.js

```diff
@@ -48,9 +48,21 @@ async function init(fastly, version, options, config) {
     })
     : include(vclfile);
   content = regex(content, urlFilters);
-  content = injectConsts(content, {
+
+  // create the constants object
+  const constants = {
     preflight: config.preflight ? config.preflight.replace('https://adobeioruntime.net', '') : undefined,
+  };
+
+  // define default version constants
+  const defVersions = {
+    ...defaultVersions,
+    ...versions,
+  };
+  Object.values(defVersions).forEach((k, v) => {
```

Comment:
I'd use `reduce` over `forEach`

Reply:
why? it only makes it look more complicated